components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +157,548 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
+Use this HTTP request to replace the policy's membership.
 
-```bash
-curl -s -H "api-token: $TOKEN" -X POST -d '{"members":["team:local:admins"]}' https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}/members:remove?pretty
-```
+### Adding Members to a Policy
 
-In the browser: Navigate to the *Policies* page under the **Settings** tab., and then select an individual policy to open its detail view.
-Select its **Members** tab to view the current membership.
-Identify the member you wish to remove from the policy, open the menu at the end of its row, and then select the **Remove Member** option.
+Specify new members to add to a policy's membership via this HTTP request.
 
-### Deleting a Policy
+*In the browser:* **Settings**  >> **Policies** >> [select a policy] >> **Members** >> **Add Members**
 
-In the IAM v2 beta, policies must be deleted from the command line.
+Select any local users or teams listed to add to the policy, or use **Add Member Expressions** for tokens, or LDAP or SAML users or teams.
 
-See [Listing Policies]({{< relref "iam-v2-api-reference.md#listing-policies" >}}) to fetch a list of all your policies and locating the relevant policy ID.
-Use the policy ID as the final path component to delete a policy:
+### Removing Members from a Policy
 
-```bash
-curl -s -H "api-token: $TOKEN" -X DELETE https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}?pretty
-```
+Specify members to remove from a policy's membership via this HTTP request.
+The removed members still exists within Chef Automate, but are no longer associated with this policy.
 
-In the browser: Navigate to the *Policies* page under the **Settings** tab., and then find the policy you wish to delete.
-Use the menu at the end of the policy row and select the **Delete Policy** option to delete the policy.
+*In the browser:* **Settings**  >> **Policies** >> [select a policy] >> **Members** >> [open control menu for target member] >> Remove Member
 
 ## Roles
 
 A role encapsulates a list of actions.
-IAM v2 provides several default Chef-managed roles, which are essential to the operation of Chef Automate and cannot be altered.
-Roles you create are Custom roles.
+IAM v2 provides several default *Chef-managed* roles, which are essential to the operation of Chef Automate and cannot be altered.
+Roles you create are *Custom* roles.
+
+Method | HTTP request              | Description
+-------|---------------------------|------------
+list   | GET /roles                | lists both *Chef-managed* and *Custom* roles
+get    | GET /roles/***id***       | gets the specified role
+create | POST /roles               | creates a *Custom* role
+update | PUT /roles/***id***       | updates a *Custom* role
+delete | DELETE /roles/***id***    | deletes a *Custom* role
+
+**Default Chef-managed Roles:**
```

Comment:
why arent these headings?

Reply:
That was a deliberate choice. To me, it confuses the hierarchy. Right now, each item that is a heading (and therefore each item in the table of contents) is an endpoint. I thought this would clutter unnecessarily.